app/scripts/modules/netflix/pagerDuty/pagerDutyTag.component.ts

```diff
@@ -1,33 +1,60 @@
-import {module} from 'angular';
-import {PAGER_DUTY_READ_SERVICE, IPagerDutyService} from './pagerDuty.read.service';
+import {Component, Input, OnChanges, OnInit, SimpleChanges} from '@angular/core';
 
-export class PagerDutyTagController implements ng.IComponentController {
-  public currentService: IPagerDutyService;
+import {IDowngradeItem} from 'core/domain/IDowngradeItem';
+import {API_SERVICE_PROVIDER} from 'core/api/api.service';
+import {IPagerDutyService, PagerDutyReader} from './pagerDuty.read.service';
+
+@Component({
+  selector: 'deck-pager-duty-tag',
+  providers: [API_SERVICE_PROVIDER, PagerDutyReader],
+  template: `
+    <span>
+      <span *ngIf="!servicesLoaded">
+        <i class="fa fa-asterisk fa-spin fa-fw"></i> Loading...
+      </span>
+      <span *ngIf="servicesLoaded  && currentService">
+        {{ currentService.name }} ({{ currentService.integration_key }})
+      </span>
+      <span *ngIf="servicesLoaded && !currentService">
+        Unable to locate PagerDuty key
```

Comment:
Maybe include the key that couldn't be found, e.g. `Unable to locate PagerDuty key ({{apiKey}})`?